docs/components/inputMapper.base.md

```diff
@@ -0,0 +1,300 @@
+- `oldInputValue`: Previous value of the gamepad input before its state is changed.
+- `homepageURL`: URL that a new tab or window should load when opened.
+
+The invokers use only those arguments required to perform the navigation it's meant for. Apart from the `inputValue` and
+`oldInputValue` arguments, the remaining arguments derive their value from the configuration map.
+
+### `{inputMapper.base}.clearIntervalRecords(records)`
+
+- `records {Object}` The `intervalRecords` member containing a list of interval loops.
+- Returns: Nothing.
+
+Clears all the running interval loops when the gamepad is disconnected or when the instance of the `inputMapper.base`
+component is destroyed.
+
+### `{inputMapper.base}.tabindexSortFilter(elementOne, elementTwo)`
+
+- `elementOne {Object}` The DOM element to be compared and sorted.
+- `elementTwo {Object}` The DOM element to be compared and sorted.
+- Returns: `Integer` An integer determining the order of two elements.
+
+Sorts all the tabbable elements in the order of their `tabindex` value. If none of the elements have a `tabindex`
+attribute then the elements are placed in the given order.
+
+### `{inputMapper.base}.trackDOM(that)`
+
+- `that {Object}` The `inputMapper.base` component.
+- Returns: Nothing.
+
+Listens for changes to the DOM using `MutationObserver`. When the DOM has loaded or is changed, it creates a list of
+tabbable elements, sorts them using
+[`{inputMapper.base}.tabindexSortFilter`](#inputmapperbasetabindexsortfilterelementone-elementtwo), and then stores them
+the `tabbableElements` member variable for use during tab navigation.
+
+### `{inputMapper.base}.stopTrackingDOM()`
+
+- Returns: Nothing.
+
+Stops the `MutationObserver` called by the [`{inputMapper.base}.trackDOM`](#inputmapperbasetrackdomthat) when the
+instance of the `inputMapper.base` component is destroyed. It avoids listening changes to the DOM any further.
+
+## Navigation Invokers
+
+### `{inputMapper.base}.click(value)`
+
+- `value {Number}` Current value of the gamepad input.
+- Returns: Nothing.
+
+Performs **click** on the focused element when the gamepad `button` or `trigger` is pressed. However, the `select`
+dropdown is toggled by changing its `size` attribute when clicked using the gamepad.
+
+### `{inputMapper.base}.previousPageInHistory(that, value)`
+
+- `that {Object}` The `inputMapper.base` component.
+- `value {Number}` Current value of the gamepad input.
+- Returns: Nothing.
+
+Navigates to the previous page in history when the gamepad `button` or `trigger` is pressed. It uses the
+[`History API`](https://developer.mozilla.org/en-US/docs/Web/API/History_API) for the core functionality. The `focused
+element` of current web page is saved using [`chrome.storage`](https://developer.chrome.com/extensions/storage) before
+the operation is performed. It is used for restoring focus on the same element when the user navigates back to the same
```

Comment:
I would spell out "it" here, i.e. "the saved value is used to restore focus" or something like that.

Reply:
"focused element" probably shouldn't be bold.

Reply:
Sorry, I just noticed the backticks around "chrome.storage" and "History API".  We should reserve backticks for variables, gradeNames, invokers, i.e. things referenced in actual code.

Reply:
Sorry for putting those backticks on the History API. But I think we should put those on "chrome.storage" as we are using it in the same way, i.e. `chrome.storage.local.get`, `chrome.storage.local.set`, et cetera.

Reply:
Sounds good.